webpack.v8.config.js

```diff
@@ -0,0 +1,17 @@
+let path = require('path')
+module.exports = {
+  entry: path.join(__dirname, "v8env", "index"),
+  devtool: 'inline-source-map',
```

Comment:
The v8env changes are not compatible with master: You'll want to generate a normal `source-map` (not `inline-source-map`)